src/backend/AuthService/BusinessLogic/UserSettings/UserSettingsAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AuthService.BusinessLogic.Exceptions;
+using AuthService.BusinessLogic.PostLogin;
+using Core.Auth0;
+using Core.Database;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AuthService.BusinessLogic.UserSettings
+{
+    public class UserSettingsAction
+    {
+        private readonly ICurrentUserService _currentUserService;
+        private readonly UserSettingsService _userSettingsService;
+        private readonly UserDataValidationService _userDataValidationService;
+        private readonly AppDbContext _appDbContext;
+
+        public UserSettingsAction(AppDbContext appDbContext, ICurrentUserService currentUserService, UserSettingsService userSettingsService, UserDataValidationService userDataValidationService)
```

Comment:
Jak jest wiele dependencji to jest poenteruj, np.:
```
...
MyService1 myService1,
MyService2 myService2,
...
```

Reply:
usunąłem appDbContext z tego, i nie jest już tak dużo więc zostawiłem w jednej linii

Reply:
Poprawiłem, ale sprawdź czy o taki sposób Ci chodziło ;d bo różne wersję widziałem i nie mogłem się zdecydować xd